examples/Remarketing/AddFlightsFeed.cs

```diff
@@ -0,0 +1,411 @@
+                    case "Flight Description":
+                        feedAttributes[FlightPlaceholderField.FlightDescription] = feedAttribute;
+                        break;
+
+                    case "Destination ID":
+                        feedAttributes[FlightPlaceholderField.DestinationId] = feedAttribute;
+                        break;
+
+                    case "Flight Price":
+                        feedAttributes[FlightPlaceholderField.FlightPrice] = feedAttribute;
+                        break;
+
+                    case "Flight Sale Price":
+                        feedAttributes[FlightPlaceholderField.FlightSalePrice] = feedAttribute;
+                        break;
+
+                    case "Final URLs":
+                        feedAttributes[FlightPlaceholderField.FinalUrls] = feedAttribute;
+                        break;
+                    // The full list of FlightPlaceholderFields can be found here
+                    // https://developers.google.com/google-ads/api/reference/rpc/google.ads.googleads.v2.enums#flightplaceholderfieldenum.
+                    default:
+                        throw new Exception("Invalid attribute name.");
+                }
+            }
+            return feedAttributes;
+        }
+
+        /// <summary>
+        /// Creates a feed mapping for a given feed.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the flights feed is
+        /// added.</param>
+        /// <param name="feedAttributes">The feed attributes.</param>
+        /// <param name="feedResourceName">The resource name of the feed.</param>
+        private void CreateFeedMapping(GoogleAdsClient client, long customerId,
+            Dictionary<FlightPlaceholderField, FeedAttribute> feedAttributes,
+            string feedResourceName)
+        {
+            // Get the FeedMappingServiceClient.
+            FeedMappingServiceClient feedMappingService = client.GetService(
+                Services.V2.FeedMappingService);
+
+            // Maps the FeedAttributeIds to the fieldId constants.
+            AttributeFieldMapping flightDescriptionMapping = new AttributeFieldMapping()
+            {
+                FeedAttributeId = feedAttributes[FlightPlaceholderField.FlightDescription].Id,
+                FlightField = FlightPlaceholderField.FlightDescription
+            };
+
+            AttributeFieldMapping destinationIdMapping = new AttributeFieldMapping()
+            {
+                FeedAttributeId = feedAttributes[FlightPlaceholderField.DestinationId].Id,
+                FlightField = FlightPlaceholderField.DestinationId
+            };
+
+            AttributeFieldMapping flightPriceMapping = new AttributeFieldMapping()
+            {
+                FeedAttributeId = feedAttributes[FlightPlaceholderField.FlightPrice].Id,
+                FlightField = FlightPlaceholderField.DestinationId
```

Comment:
Is this supposed to be FlightPlaceholderField.FlightPrice?

Reply:
Done.